### PR TITLE
Added some books and section

### PR DESCRIPTION
**Pull request by alinadir44**
## What does this PR do?
Add resources

### Description
* [Programming in Lua (fourth edition)] https://moys.gov.iq/upload/common/Programming_in_Lua%2C_4th_ed._%282017%29_.pdf - Roberto I. (PDF)
* [bookdown: Authoring Books and Technical Documents with R Markdown] (https://bookdown.org/yihui/bookdown/) - Yihui Xie - bookdown
* [YAML Tutorial] (https://www.tutorialspoint.com/yaml/yaml_tutorial.pdf) - TutorialsPoint (PDF)
### Why is this valuable (or not)?
* Updated book added for Lua Langauge
* YAML language section added which is crucial for writing Ansible playbooks for DevOps
### How do we know it's really free?
* The provided URLs are from the publisher's official website, and so are free to view
### For book lists, is it a book? For course lists, is it a course? etc.
* All resources added are books
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1483,6 +1483,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Lua Programming](https://en.wikibooks.org/wiki/Lua_Programming) - Wikibooks
 * [Lua Tutorial](http://www.tutorialspoint.com/lua/) - Tutorials Point (HTML, PDF)
 * [Programming in Lua (first edition)](https://www.lua.org/pil/contents.html)
+* [Programming in Lua (fourth edition)](https://moys.gov.iq/upload/common/Programming_in_Lua%2C_4th_ed._%282017%29_.pdf) - Roberto Ierusalimschy (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1486]: This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.

Comment (eshellman) [reply books/free-programming-books-langs.md:1486]: remove it

**Review by eshellman — CHANGES_REQUESTED** [review]

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (alinadir44) [reply books/free-programming-books-langs.md]: removecd the reaquested Lua book. Please check now.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**